test/quantization/test_quantize_jit.py

```diff
@@ -2506,20 +2506,22 @@ def forward(self, x):
         for qconfig in [float16_dynamic_qconfig, default_dynamic_qconfig]:
             m = prepare_dynamic_jit(model, {'': qconfig})
 
-            # for input of FC for dynamic quant
-            assert len(attrs_with_prefix(m, '_observer_')) == 1
-            # for weight
+            # observer for weight
             assert len(attrs_with_prefix(m.fc, '_observer_')) == 1
+
             if qconfig == float16_dynamic_qconfig:
-                observer_name = 'NoopObserver = prim::GetAttr[name="_observer_'
+                observer_name = 'Fp16Observer = prim::GetAttr[name="_observer_'
+                FileCheck().check(observer_name) \
+                           .run(m.fc.graph)
```

Comment:
Looks like this check is not very useful, what do we want to check here?

Reply:
Just additional check for observer name match